src/Machine.Specifications.Runner.VisualStudio/Machine.Specifications.Runner.VisualStudio.csproj

```diff
@@ -22,19 +22,24 @@
 
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="[0.10-*, 0.11)" />
-    <PackageReference Include="Machine.Specifications" Version="[0.11.0, 1.0.0]" PrivateAssets="All" />
+    <PackageReference Include="Machine.Specifications" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'">
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="[15.0.0,17.0.0)" PrivateAssets="All" />
   </ItemGroup>
-  
+
   <ItemGroup Condition="'$(TargetFramework)'=='net46'">
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="[11.0.0,17.0.0)" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='net452'">
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="[11.0,12.0]" PrivateAssets="All" />
```

Comment:
Version range should be the same as net46. As above, lets just use net452 instead of net46.

Reply:
The reason i did this is it looks like net46 CAN have a later version

Reply:
Ah I see now, yep this should be fine.